packages/nimble-components/src/wafer-map/tests/wafer-map-matrix.stories.ts

```diff
@@ -115,6 +117,21 @@ const componentWaferWithDieOrientation = ([
 >
 </nimble-wafer-map>`;
 
+const componentWaferResize = (
+    size: number
+): ViewTemplate => html`<nimble-wafer-map
+        class="size${size}"
+        :dies="${() => waferMapDie}"
+        :colorScale="${() => defaultColor}"
+    >
+    </nimble-wafer-map>
+    <style>
+        nimble-wafer-map.size${size} {
+            width: ${size}px;
+            height: ${size}px;
+        }
+    </style> `;
+

```

Comment:
Looking at the example in devtools one thing I notice is that color-scale-mode binding is broken:
![image](https://user-images.githubusercontent.com/1588923/211611122-ca1d6139-5fa4-4148-b4d5-aea9be58e8e1.png)

The issue is that the color scale mode attribute decorator got pushed to the wrong property:
![image](https://user-images.githubusercontent.com/1588923/211611538-3392f9b6-d35b-4d3a-b706-7713c49b1d67.png)

As the source screenshot shows it is currently bound to the canvas element which explains the dev tools serialization.

Instead it should be bound to the colorScaleMode property:

```ts
    @attr({
        attribute: 'color-scale-mode'
    })
public colorScaleMode: WaferMapColorScaleMode = WaferMapColorScaleMode.linear;
```

That could either be addressed in this PR or a follow-up.